raster-foundry/raster-foundry
# Use scalafix to remove a ton of things we don't need

## Overview

This PR removes a bunch of things that weren't in use using scalafix and some manual work.

### Checklist


### Notes

Some of the things here were manual -- for example, scalafix didn't seem to want to remove unused values bound in for comprehensions. It also does a weird thing with unused private vals where it removes the assignment... but still does whatever computation was being assigned. I think this is because it can't tell from a return type whether there are important side effects happening, but it's still pretty funny looking. Anyway, in cases where that happened, I just deleted things.

There's also a [`scalafix --check`](https://scalacenter.github.io/scalafix/docs/users/installation.html#enforce-in-ci) command that we can add an alias for -- unclear whether we want that though

## Testing Instructions

 * use the app normally and confirm everything is fine -- get some lab tiles, view some projects, look at some datasources... you know, be like a user

Closes #4500
